scripts/docker-helper-4.x/cht-docker-compose.sh

```diff
@@ -261,23 +262,45 @@ get_system_and_docker_info(){
   echo "---DEBUG INFO---"
   echo "Load: $(get_load_avg)"
   echo "CHT Containers: $(get_running_container_count)"
-  echo "Global Containers $(get_global_running_container_count)"
+  echo "Global Containers: $(get_global_running_container_count)"
+  echo "URL: $projectURL"
   echo
   echo $"$info" | column -t
 }
 
+update_nginx_local_ip_tls_cert(){
+  nginxContainerId=$1
+  rm -f /tmp/local-ip-fullchain /tmp/local-ip-key
+  curl --fail --silent --show-error -o /tmp/local-ip-fullchain https://local-ip.medicmobile.org/fullchain
+  curl --fail --silent --show-error -o /tmp/local-ip-key https://local-ip.medicmobile.org/key
+  docker cp /tmp/local-ip-fullchain "${nginxContainerId}":/etc/nginx/private/cert.pem  2>/dev/null
+  docker cp /tmp/local-ip-key "${nginxContainerId}":/etc/nginx/private/key.pem  2>/dev/null
+  docker exec "$nginxContainerId" bash -c "nginx -s reload"  2>/dev/null
+}
+
+validate_tls(){
+  url=$1
+  # by default curl validates TLS.  If we get back  60 then TLS isn't valid:
+  # exitcode: https://everything.curl.dev/usingcurl/verbose/writeout.html
+  # 60: https://everything.curl.dev/cmdline/exitcode.html
+  status=$(curl  --write-out "%{exitcode}"  -qs  "$url" -o /dev/null)
```

Comment:
Thanks for the command explanation

Reply:
curl is powerful, but can be a bit obtuse!